apps/server/lib/lexical/server/state.ex

```diff
@@ -66,6 +67,7 @@ defmodule Lexical.Server.State do
 
   def apply(%__MODULE__{shutdown_received?: true} = state, %Exit{}) do
     Logger.warn("Received an Exit notification. Halting the server in 150ms")
+    RemoteControl.halt(state.configuration.project)
```

Comment:
the deadline on `RemoteControl.halt` is over five seconds now, but the prior deadline was 50ms.